amperity_datagrid/source/bridge_snowflake.rst

```diff
@@ -0,0 +1,346 @@
+
+A sync from Snowflake to Amperity requires configuration steps to be made in both Amperity and Snowflake.
+
+.. bridge-snowflake-sync-amperity-end
+
+.. bridge-snowflake-sync-amperity-links-start
+
+#. :ref:`Get details <bridge-snowflake-sync-amperity-get-details>`
+#. :ref:`Configure Snowflake <bridge-snowflake-sync-amperity-configure-snowflake>`
+#. :ref:`Add inbound bridge <bridge-snowflake-sync-amperity-add-bridge>`
+
+.. bridge-snowflake-sync-amperity-links-end
+
+
+.. _bridge-snowflake-sync-amperity-get-details:
+
+Get details
+--------------------------------------------------
+
+.. bridge-snowflake-sync-amperity-get-details-start
+
+Before you can create inbound sharing between Snowflake and Amperity you will need to collect the following information.
+
+.. list-table::
+   :widths: 10 90
+   :header-rows: 0
+
+   * - .. image:: ../../images/steps-arrow-off-black.png
+          :width: 60 px
+          :alt: Requirement 1.
+          :align: left
+          :class: no-scaled-link
+     - The organization name and account name for your brand's |destination-name| account.
+
+       .. include:: ../../shared/credentials_settings.rst
+          :start-after: .. credential-snowflake-organization-start
+          :end-before: .. credential-snowflake-organization-end
+
+       .. include:: ../../shared/credentials_settings.rst
+          :start-after: .. credential-snowflake-account-name-start
+          :end-before: .. credential-snowflake-account-name-end
+
+       .. tip:: To `find your organization name and account name <https://docs.snowflake.com/en/user-guide/admin-account-identifier#finding-the-organization-and-account-name-for-an-account>`__ |ext_link| use **Snowsight**.
+
+          Open the account selector and browse to the account for which Amperity Bridge will be configured. Hover over the account name to view additional details, and then copy the account identifier.
+
+          The copied identifier contains both organization and account name in the format `organization-name.account-name`.
+
+
+   * - .. image:: ../../images/steps-arrow-off-black.png
+          :width: 60 px
+          :alt: Requirement 2.
+          :align: left
+          :class: no-scaled-link
+     - .. include:: ../../shared/credentials_settings.rst
+          :start-after: .. credential-snowflake-region-start
+          :end-before: .. credential-snowflake-region-end
+
+
+   * - .. image:: ../../images/steps-arrow-off-black.png
+          :width: 60 px
+          :alt: Requirement 3.
+          :align: left
+          :class: no-scaled-link
+     - Before you can configure Amperity Bridge for data sharing with Snowflake you must :ref:`create a share in Snowflake <bridge-snowflake-sync-amperity-configure-snowflake-create-share>`, add a secure share identifier to that share, identify the correct account locator, and then add the :ref:`Amperity account locator ID <bridge-snowflake-sync-amperity-configure-snowflake-account-locator>` to the share.
```

Comment:
Before you can configure Amperity Bridge for data sharing with Snowflake you mustcreate a share in Snowflake, add a secure share identifier to that share, identify the region for your Snowflake account, and add the locator ID for the Amperity account in your region as a consumer of the share.

Use the [CURRENT_REGION](https://docs.snowflake.com/en/sql-reference/functions/current_region) function to return the name of the region for the account where the current user is logged in.